src/components/EventCard/EventCard.tsx

```diff
@@ -88,14 +79,23 @@ const EventCard = ({ content }: IEventCardProps) => {
             <span className="font-bold text-primary">+3 other</span>
           </span>
         </div>
-
-        <p
-          onClick={toggleShowMoreEventDetails}
-          tabIndex={0}
-          className="mt-4 cursor-pointer text-sm font-bold text-primary underline focus-visible:outline-focus lg:text-base"
-        >
-          {showMoreEventDetails ? "Show less details" : "Show more details"}
-        </p>
+        <div className="mt-4 hidden justify-between gap-x-3 align-bottom text-base text-primary underline lg:flex">
+          <p
```

Comment:
This should be button. Because 'Show more' or 'Show less' actually has click functionality. Please change the semantics to `button` and remove `tabIndex`